package.json

```diff
@@ -36,11 +36,17 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run lint",
+      "pre-commit": "lint-staged",
       "pre-push": "npm run test",
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
   },
+  "lint-staged": {
+    "linters": {
+      "*.ts": "ng lint --files"
```

Comment:
Great to see this change! If we change this to:
```
["*.ts": "ng lint --fix --files",
"git add"]
```

We also get the changes for my lint that could be fixed in the commit. If it errors it still doesn't commit files.

Reply:
Gotcha.